docs/topics/constructing-collections.md

```diff
@@ -31,6 +31,24 @@ initialization fluent here.
 val numbersMap = mutableMapOf<String, String>().apply { this["one"] = "1"; this["two"] = "2" }
 ```
 
+## Create with collection builder functions
+
+Another way of creating a collection is to call a builder function –
+[`buildList()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/build-list.html), [`buildSet()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/build-set.html),
+or [`buildMap()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/build-map.html). They create a new
```

Comment:
I feel awkward whenever I use pronouns for such objects — functions, classes, etc.
I prefer to use "These functions create". But maybe it looks redundant. 
WDYT?

Reply:
I think it's OK to use such pronouns even with inanimate objects. It looks simpler and more colloquial
I can double-check with copyeditors